vyxal/vyxal.py

```diff
@@ -0,0 +1,21 @@
+"""
+File: vyxal.py
+Description: This is the main file for the project - the transpilation
+of Vyxal programs actually happens here, and this is what gets executed
+offline.
+"""
+
+from vyxal.context import Context
+from vyxal.transpile import transpile
+
+ctx = Context()
```

Comment:
does this need to be passed in somewhere?

Reply:
This will be passed between functions in the transpiled program.